news: Accept first customers (#71)
flowforge/website: src/blog/flowforge-accepting-customers.md

---
title: FlowForge is accepting customers now
subtitle: From today forward, we start accepting customers of our waitlist
description: From today forward, we start accepting customers of our waitlist
date: 2022-04-04
authors: ["zeger-jan-van-de-weg"]
---

> A year ago our CTO, Nick O'Leary, [introduced FLowForge](https://flowforge.com/blog/2021/04/first-deploy/).

Reviewer: ```suggestion
A year ago our CTO, Nick O'Leary, [introduced FlowForge](https://flowforge.com/blog/2021/04/first-deploy/).
```

Since then major milestones have been achieved. As we grow as a company, important
steps are taken. Today we make another very important step; we're accepting our
first customers.

<!--more-->

A few weeks ago our product was nearly in a state where it could support customers,
when that was the case, we [opened up our waitlist](...). The waitlist is growing

> daily, a trend we're aiming to break today. The first lucky few users are added
> today, and each working day for the foreseeable future new ones are added. New

Reviewer: ```suggestion
daily, a trend we're aiming to break today. The first lucky few users are being added
today, and each working day for the foreseeable future we'll continue onboarding users. New
```

users will receive an email from our team with their login details and can start
creating new workflows with Node-RED minutes after.

While the source code of FlowForge is [available](https://github.com/flowforge/flowforge),
there's a chance you're unfamiliar with what has been built around Node-RED.
With FlowForge, our intent is to build a platform to aid with colaboration of

> flows for any Node-RED. The first steps to our vision include a managed

Reviewer 2: ```suggestion
flows in Node-RED. The first steps to our vision include a managed
```

Node-RED instance to connect virtually any online service with another with the

> interface already knows. Multiple users will have access to the same flows and

Reviewer 2: Delete `with the
interface already knows.`
It doesn't make sense

> can collaborate. Further, once Node-RED 3.0 will be release, upgrading will take

Reviewer 2: ```suggestion
can collaborate. Further, once Node-RED 3.0 has been released, upgrading will take
```

only a single click. There's many more exciting features right around the corner

> on our [roadmap](https://github.com/orgs/flowforge/projects/5).

Reviewer 2: Roadmap isn't currently public, but I'm happy to open it up

Author: Didn't consider that, but would be nice to open up.
